orchestrator/workflow.py

```diff
@@ -92,7 +92,7 @@ class Workflow(Protocol):
     name: str
```

Comment:
Target is still not shown correctly on task list page, checking out why

Reply:
This is not shown correctly because of old incorrect database structure, created a issue to fix this: https://github.com/workfloworchestrator/orchestrator-core/issues/409